src/apps/components/AppDetailsPage/AppDetailsPage.tsx

```diff
@@ -0,0 +1,195 @@
+import Button from "@material-ui/core/Button";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+import AppHeader from "@saleor/components/AppHeader";
+import CardSpacer from "@saleor/components/CardSpacer";
+import CardTitle from "@saleor/components/CardTitle";
+import Container from "@saleor/components/Container";
+import ExternalLink from "@saleor/components/ExternalLink";
+import PageHeader from "@saleor/components/PageHeader";
+import Skeleton from "@saleor/components/Skeleton";
+import { sectionNames } from "@saleor/intl";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import ReactMarkdown from "react-markdown";
+
+import activateIcon from "../../../../assets/images/activate-icon.svg";
```

Comment:
There's an alias for this

```suggestion
import activateIcon from "@assets/images/activate-icon.svg";
```